src/v8.2/refactor-comet-gen-field-properties.ts

```diff
@@ -0,0 +1,28 @@
+import { promises as fs } from "fs";
+import { glob } from "glob";
+
+/**
+ * Refactors field properties in cometGen.ts / cometGen.tsx files:
+ * - gqlName -> rootQueryArg
+ * - fieldName: -> formFieldName:
+ */
+export default async function refactorCometGenFieldProperties() {
+    console.log("Try to run");
+    const files: string[] = glob.sync("**/*cometGen.{ts,tsx}");
+
+    for (const filePath of files) {
+        let content = await fs.readFile(filePath, "utf-8");
+        const originalContent = content;
+
+        // Replace gqlName -> rootQueryArg
+        content = content.replace(/\bgqlName\b/g, "rootQueryArg");
+
+        // Replace object property name: -> formFieldName:
+        content = content.replace(/\bfieldName\b:/g, "formFieldName:");
```

Comment:
This changes any occurrences of gqlName and fieldName, not just those inside asyncSelect, which could have side effects. Can we ignore this?

Reply:
Maybe we could change the codemod to a more sophisticated one using ts-morph?